src/swarm/neo/client/request_options/RequestContext.d

```diff
@@ -165,7 +165,7 @@ public struct RequestContext
 
     ***************************************************************************/
 
-    public Object object ( )
+    public Object object ( ) /* d1to2fix_inject: const */
```

Comment:
The core of my problem: strictly speaking this is a breaking change...

Reply:
How does this break the library API?

Reply:
Ha, this is interesting - it doesn't! The reason why I thought it would break API is I assumed that the `Object` returned would be `const Object` (since it's the part of the `RequestContext` object, and this method is marked `const`, so the `RequestContext` should not change). However, there's a carefuly placed `cast` here, so this all works (although not sure how correct it is).

In other words:

```
class X
{
    import ocean.core.ContextUnion;

    private ubyte[ContextUnion.sizeof] context_;
    private Object o;

    Object get() const // compiles
    {
        return (cast(ContextUnion*)this.context_.ptr).object();
    }

    Object getMe() const // doesn't compile - can't cast this.o of type `const(Object)` to `Object`
    {
        return this.o;
    }
}
```

Reply:
Ha, interesting. All good then.